assets/js/lesson_3/calculator.js

```diff
@@ -0,0 +1,34 @@
+const userAction = prompt("Choose what you want to do: 'add', 'sub', 'mult', 'div'")
+
+if (userAction === 'add' || userAction === 'sub' || userAction === 'mult' || userAction === 'div') {
+    const firstNum = +prompt('Enter first number');
+    const secondNum = +prompt('Enter second number');
+
+    if (!isNaN(firstNum) && !isNaN(secondNum)) {
+        const add = firstNum + secondNum;
+        const sub = firstNum - secondNum;
+        const mult = firstNum * secondNum;
+        const div = firstNum / secondNum;
```

Comment:
this is a great idea to move all calculations to variables, but this is overhead. You need ONLY one calculation before alert